packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -119,7 +112,7 @@ const sizeStyles = (props: ThemeProps<DefaultTheme>) => {
 export const StyledGlobalAlert = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
-})`
+})<IStyledGlobalAlertProps>`
```

Comment:
Is this type parameter required?

Reply:
Just so the `$alertType` prop is valid in the `elements` JSX side.